### PR TITLE
home page

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added home page
 - Changed fonts from font awesome to google fonts
 
## Files not to be review:
- images
- yarn lock
- package-lock.json

## Link:
https://deploy-preview-2--evolt-ecommerce.netlify.app/

## Screenshot:
image
image

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Footer.css";
+
+export const Footer = () => {
+	return (
+		<footer className="footer flex-column flex-center p-4">
+			<p className="mb-1">Made by Rohan</p>
+
+			<div>
+				<a
```
Comment (ishitakeshawani) [inline src/components/Footer/Footer.jsx:10]: Maybe you can use Link here.

**Review by ishitakeshawani — APPROVED** [review]

**Review by prakashsakari — COMMENTED** [review]
1. UI look neat and clean
2. Easy to find available options
3. You can remove comment before pushing.

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (rohanmathur91) [reply src/components/Footer/Footer.jsx:10]: Links can be used inside our apps only for, external things we need anchor tags.

**Review by prakashsakari — APPROVED** [review]
Added some feedback you can have a look at them

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (ishitakeshawani) [reply src/components/Footer/Footer.jsx:10]: ok

File: src/components/Category/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		<>
+			<div className="text-base font-semibold flex-row content-space-between mx-5 gutter-bottom">
+				<div className="category-header font-semibold">Categories</div>
+				<Link to={"/products"} className="flex-row flex-center">
+					<span className="category-header-btn font-semibold">Shop now</span>{" "}
+					<span class="material-icons-outlined font-base ml-1">east</span>
+				</Link>
+			</div>
+			<article className="category">
+				{categories.map(({ alt, image, category }, index) => (
```
Comment (ishitakeshawani) [inline src/components/Category/Category.jsx:17]: There is `/api/categories` route. Don't we need to use it here and add category information in db?

Comment (rohanmathur91) [reply src/components/Category/Category.jsx:17]: I will add in the next PR.

**Pull request merged**